src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -76,14 +76,23 @@ export default class EthereumBlockchainService implements BlockchainService {
     } else {
       const gasPriceEstimate = await this.provider.getGasPrice();
```

Comment:
Can we add a comment or something here for the units? I think this gives back the amount in "wei".

Reply:
I'd actually rather not since it's not strictly relevant to our code and is really a property of the ethers library.  I don't feel like we should be trying too hard to document the behavior of libraries we depend on, or else we risk getting out of sync with them.  For example, what if over time deflation causes ethers to change what unit is returned from this function, and now our comments are out of date?

Reply:
To do math properly we need to know the units though. And when we pass the value to a transaction we need to ensure it is wei not gwei, independent of what library we use.

It would be great if we had types for the denominations but I haven't seen a library that does that yet.

Reply:
> To do math properly we need to know the units though

How so?  10% is 10% regardless of units.

> when we pass the value to a transaction we need to ensure it is wei not gwei, independent of what library we use.

I would assume that if we're using the `ethers` library for everything, then as long as it's internally consistent we can pass around whatever values it gives us without needing to know or care what the underlying units are

Reply:
Yeah I strongly dislike the assumption and I'm not positive it will be true over time. But I'm fine if you don't want to add a comment bc agree that could get stale in the future. Would push to include it in debug log though plz.

Reply:
Alright, I'm still worried about enshrining assumptions about upstream packages into our code, but I don't feel *that* strongly about it, so I'll go ahead and add the comment and log message for now